slack/README.md

```diff
@@ -0,0 +1,63 @@
+# smarthome-slack
+Plugin to send push notifications to Slack 
+
+## Installation
+<pre>
+cd smarthome.py directory
+cd plugins
+git clone https://github.com/rthill/slack.git
+</pre>
```

Comment:
Do we need some installation instructions? Usually the plugins repos is checked out completely in a given version.

But if we need, can you change the repo URL to point to the SHNG repository?